contracts/LockedFund/LockedFund.sol

```diff
@@ -180,7 +221,8 @@ contract LockedFund is ILockedFund {
 
 		waitedTS = _waitedTS;
 		token = IERC20(_token);
-		vestingRegistry = IVestingRegistry(_vestingRegistry);
+		vestingRegistry = IVestingRegistryLogic(_vestingRegistry);
+		vestingCreationType = 1;
 
 		for (uint256 index = 0; index < _admins.length; index++) {
 			require(_admins[index] != address(0), "LockedFund: Invalid Address.");
```

Comment:
consider checking the address is contract e.g. using OZ library Address.sol

Reply:
Solved in c47466e